src/reportWebVitals.js

```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
       getCLS(onPerfEntry);
       getFID(onPerfEntry);
       getFCP(onPerfEntry);
```

Comment:
Good job so far. You've created an app and added the required dependencies like `big.js.` and `prop-types` :+1: 

- [ ] You still have unnecessary files like this in your repo. e.g `logo.svg` and others. Any file that will not be of direct usage in your app should be deleted.

Reply:
Got it!